draft-irtf-cfrg-opaque.md

```diff
@@ -1330,11 +1316,58 @@ Derive-Secret(Secret, Label, Transcript-Hash) =
 
 Note that the Label parameter is not a NULL-terminated string.
 
+OPAQUE-3DH integrates configuration and additional information
```

Comment:
I would simplify this and just say that `context` is an arbitrary-length byte string, and it's encoded in the transcript preamble like so:

```
preamble = concat(I2OSP(len(context), 2), context, ...
```

We could give guidance for what this context should include, such as what you list below, but it _seems_ overly prescriptive to force this particular structure here.